src/server/migrations/20200325162313_crop_stages.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('crop_stages', function(table) {
+    table
+      .increments('id')
+      .primary()
+      .unsigned();
+    table.enu('name', []).notNullable();
```

Comment:
I am not sure with this syntax, Daniel please confirm this please

Reply:
lets see what Daniel suggests.

Reply:
@amjadiftikhar if you need @dpfernandes help to remember to tag him otherwise he will not see it and follow up to get the information you need 🙏 

And this syntax is fine but when using `enums` you have to specify the allowed list of `enums` not an empty array with no options

@amjadiftikhar check this 👍 
http://knexjs.org/#Schema-enum
https://www.mysqltutorial.org/mysql-enum/